launcher/ui/dialogs/ProfileSetupDialog.cpp

```diff
@@ -261,7 +264,21 @@ void ProfileSetupDialog::setupProfileFinished()
     } else {
         auto parsedError = MojangError::fromJSON(*m_profile_response);
         ui->errorLabel->setVisible(true);
-        ui->errorLabel->setText(tr("The server returned the following error:") + "\n\n" + parsedError.errorMessage);
+
+        QString errorMessage = tr("Network error(%1): %2: HTTP Status: %3")
+                                   .arg(QString::number(m_profile_task->error()), m_profile_task->errorString(),
+                                        QString::number(m_profile_task->replyStatusCode()));
```

Comment:
```suggestion
        QString errorMessage =
            tr("Network Error: %1\nHTTP Status: %2").arg(m_profile_task->errorString(), QString::number(m_profile_task->replyStatusCode()));

```
I don't think there's any point showing the error code as it's just an enum value that i don't think is standardised in any way